async_simple/coro/test/LazyTest.cpp

```diff
@@ -1177,7 +1177,16 @@ Lazy<int> lazy_fn<0>() {
 }
 }  // namespace detail
 TEST_F(LazyTest, testLazyPerf) {
```

Comment:
we already have benchmark for performance test, this case can be removed?

Reply:
More tests are never bad : )